docs/_config.yml

```diff
@@ -0,0 +1,17 @@
+title: Saros
+subtitle: Distributed Party Programming
+email: your-email@example.com
+description: >- # this means to ignore newlines until "baseurl:"
+  Write an awesome description for your new site here. You can edit this
+  line in _config.yml. It will appear in your document head meta (for
+  Google search results) and in your feed.xml site description.
```

Comment:
It seems to me like the fields email and description are still set to default values.

What is the email field used for?

Add a general description of the page? Something like "Homepage of Saros, an IDE plugin which enabled users to participate in distributed pair programming sessions."?
I found a page about how google handles these description meta tags, but I still don't know what would be best to put here. You can have a look for yourself: https://support.google.com/webmasters/answer/35624?visit_id=1-636674263008325798-3147872153&rd=1
It's the last section titled "Create good meta descriptions".